ext/com_dotnet/com_wrapper.c

```diff
@@ -508,7 +508,7 @@ static php_dispatchex *disp_constructor(zval *object)
 {
 	php_dispatchex *disp = (php_dispatchex*)CoTaskMemAlloc(sizeof(php_dispatchex));
 
-	trace("constructing a COM wrapper for PHP object %p (%s)\n", object, Z_OBJCE_P(object)->name);
+	trace("constructing a COM wrapper for PHP object %p (%s)\n", object, Z_OBJCE_P(object)->name->val);
```

Comment:
Is `name` a `zend_string`? If so I think using the usual macro is "better" as it seems to be the general approach of the rest of the codebase.

Reply:
Oh, indeed. I have mindlessly copied that from

https://github.com/php/php-src/blob/9a70b7c221df1ce83bf78306af76efee6b38f2e4/ext/com_dotnet/com_wrapper.c#L73

which had been commited by Dmitry back then, and we missed its tenth birthday only by three days. So much about the state of com_dotnet.

We do not have a macro to get the classname (instead of using `Z_OBJCE(…)->name`)?

Reply:
I can't find one, so I am going to say that we don't have one.